src/app.scss

```diff
@@ -66,3 +66,11 @@ body,
     }
   }
 }
+
+.table {
+  td:last-child {
+    button {
+      float: right;
```

Comment:
![image](https://user-images.githubusercontent.com/15377965/152414272-3c47cc98-731b-4e01-8982-1901d0798d5b.png)

 instead of 

![image](https://user-images.githubusercontent.com/15377965/152414347-062dccc4-8f80-4f3e-9d85-3fbeb9fbf293.png)